activation/activation.go

```diff
@@ -343,7 +358,7 @@ func (b *Builder) buildInitialPost(ctx context.Context) error {
 		return nil
 	}
 	// ...and if we haven't stored an initial post yet.
-	_, err := nipost.InitialPost(b.localDB, b.signer.NodeID())
+	_, err := nipost.GetPost(b.localDB, b.signer.NodeID())
```

Comment:
Getters in go usually drop the `Get-` prefix: https://go.dev/doc/effective_go#Getters

```suggestion
	_, err := nipost.Post(b.localDB, b.signer.NodeID())
```

Reply:
Sure, but the problem is that `Post` already exists in package `nipost`.

Reply:
This is related to the other point: this should really just be the initial post of a node, and for most nodes it will be. Putting another post into this table is just a fallback for some nodes that otherwise would have to regenerate it.